README.md

```diff
@@ -1,21 +1,28 @@
 # Rails Hello World
 
-This lab will create a Hello World static response
-
+In this lab you will integrate a static route so that the application will render a "Hello World" page.
 
 # Objectives
 
 1. Draw a route
 2. Map a route to a controller/action
-3. Render- implicit or explicit- a static template
+3. Render (implicitly or explicitly) a static template
 4. Create a static template
 
-## Notes
 
+## Testing
+
+The RSpec/Capybara test suite currently is not passing, to run the tests, run the command ```bundle exec rspec```, you will have passed the lab when the test is passing.
```

Comment:
By the time they get to this content, they will be very familiar with running tests! No need to include how to run them.